rcp/org.testeditor.rcp4.views.teststepselection/plugin.xml

```diff
@@ -4,10 +4,6 @@
    <extension
          id="org.testeditor.rcp4.views.teststepselector.fragment"
          point="org.eclipse.e4.workbench.model">
-      <fragment

```

Comment:
The Applicationmodel is copied to your workspace and is used. It still contains the view. If you use a new workspace for the test-editor, the view disapears.